ecgtools/parsers/cesm.py

```diff
@@ -99,7 +99,7 @@ def _join(a):
     return date
 
 
-def parse_cesm_history(file, user_streams_dict={}):
+def parse_cesm_history(file, user_streams_dict={}, xarray_open_engine='netcdf4'):
```

Comment:
Could we make this more generic to allow passing additional arguments?

```suggestion
def parse_cesm_history(file, user_streams_dict={}, xarray_open_kwargs=None):
  _default_kwargs = {'engine': 'netcdf4', chunks: {}, decode_times=False}
  if xarray_open_kwargs is None:
     xarray_open_kwargs = _default_kwargs 
   else:
     xarray_open_kwargs.update(_default_kwargs)
  
```